python_ta/reporters/node_printers.py

```diff
@@ -143,7 +150,11 @@ def render_pep8_errors(msg, _node, source_lines=None):
         error_code = matched_error.group(1)
         # Render the appropriate error through the RENDERERS dict
         if error_code in RENDERERS:
-            yield from RENDERERS[error_code](msg, _node, source_lines)
+            line = msg.line
+            col = get_col(msg)
+            yield from render_context(line - 3, line, source_lines)
+            yield from RENDERERS[error_code](msg, _node, line, col, source_lines[line - 1])
```

Comment:
Okay so looking at all of the functions, I think we can simplify even further: I don't think any of the custom rendering functions use the `msg` or `_node` arguments. Assuming this is true, we can remove them from here and from the function headers. Please double-check to see if `msg`/`_node` are used anywhere, and if not, please remove them!